setup.py

```diff
@@ -57,7 +57,10 @@
 if '.' not in sys.path:
     sys.path.append(os.getcwd() + '/src')
 
-absexpanduser = lambda x: os.path.abspath(os.path.expanduser(x))
+
+def absexpanduser(x): return os.path.abspath(os.path.expanduser(x))
```

Comment:
I am not sure, but I think it is against PEP8 to put functions on one line like this.

Reply:
my autopep8 did this one..

from https://www.python.org/dev/peps/pep-0008/ :
```
Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier.

Yes:

def f(x): return 2*x
No:

f = lambda x: 2*x
```

Reply:
Ahh that is fair :)